commands/export.go

```diff
@@ -0,0 +1,183 @@
+package commands
+
+import (
+	"context"
+	"io"
+	"os"
+
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/lily/chain/export"
+	"github.com/filecoin-project/lotus/blockstore"
+	"github.com/filecoin-project/lotus/chain/store"
+	"github.com/filecoin-project/lotus/node/repo"
+	"github.com/mitchellh/go-homedir"
+	"github.com/urfave/cli/v2"
+	"golang.org/x/xerrors"
+)
+
+type chainExportOps struct {
+	repo        string
+	max         uint64
+	min         uint64
+	outFile     string
+	progress    bool
+	includeMsgs bool
+	includeRcpt bool
+	includeStrt bool
+}
+
+var chainExportFlags chainExportOps
+
+var ExportChainCmd = &cli.Command{
+	Name:        "export",
+	Description: "Export chain from repo (requires node to be offline)",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:        "repo",
+			Usage:       "the repo to export chain from",
+			Value:       "~/.lily",
+			Destination: &chainExportFlags.repo,
+		},
+		&cli.Uint64Flag{
+			Name:        "max",
```

Comment:
`to` for consistency with walk

Reply:
ack will use to and from naming convention.